sbc_email/models/import_letters_history.py

```diff
@@ -303,10 +382,17 @@ def _manage_all_imported_files(self):
                     zip_ = zipfile.ZipFile(
                         zip_file, 'r')
 
-                    # loop over files inside zip
+                    zip_to_remove = False
                     for f in zip_.namelist():
-                        self._copy_imported_to_done_letter(
-                            f, BytesIO(zip_.read(f)), False)
+                        # when this is manual import we are not sure that this
+                        # zip contains current letters treated
+                        if not self.manual_import or self.is_in_list_letter(f):
```

Comment:
No but I am not sure how python treats condition in "if" with  "or". In Java, if first condition is true the second is not checked. 
In this case it will avoid to execute is_in_list_letter function (not useless when whre are not in manual import)